extension/extension.cpp

```diff
@@ -394,6 +413,14 @@ class UploadThread: public IThread
 
 			if (uploaded) {
 				count++;
+
+				crashIdPack *pack = new crashIdPack;
+				pack->id = count;
+				memcpy(pack->crashId, response + 10, 14);

```

Comment:
This isn't safe, the message from the server is free-form text and subject to change at any time.

Reply:
yes, I forget only to check the response if less than 14 characters... Also that I can know if you will change the Api, I can add a parse to get the ID with or without the -